### DISCUSSION TITLE
Title override for main page

### DISCUSSION THREAD

**Discussion by DannyBen** [retypeapp/retype#346] [Ideas]
I am looking for an ability to completely override the title of the main page, for search engine visibility purposes.

Right now, the main page is handled like any other page, and receiving the title suffix as defined in the config's `meta.title`:

```yaml
meta:
  title: " | Bashly - Bash CLI Framework"
```

I was looking at [retype's `title` page configuration](https://retype.com/configuration/page/#title), but it is obviously not intended for this.

Can this be done? If not, can this be considered as a feature?

Ideally, I think this should be definable at the page level:

```yaml
# page front matter
title_override: "My search-engine-friendly title"  # which will appear without the global suffix
---
```

**Comment by geoffreymcgill** [comment]
Moved this to an issue so it's easier to track as a feature request. Please see https://github.com/retypeapp/retype/issues/350.

**Reply by geoffreymcgill** [reply]
See https://github.com/retypeapp/retype/issues/350#issuecomment-1470792128
